dev/ci/ci-common.sh

```diff
@@ -107,3 +107,24 @@ install_ssreflect()
   echo -en 'travis_fold:end:ssr.install\\r'
 
 }
+
+# this installs just the ssreflect + algebra library of math-comp
+install_ssralg()
+{
+  echo 'Installing ssreflect' && echo -en 'travis_fold:start:ssr.install\\r'
+
+  git_checkout "${mathcomp_CI_BRANCH}" "${mathcomp_CI_GITURL}" "${mathcomp_CI_DIR}"
+
+  ( cd "${mathcomp_CI_DIR}/mathcomp" && \
```

Comment:
no please

Reply:
This hack was already in place and thus orthogonal to the purpose of this PR. Please let's move the discussion to its own issue  / PR.

Reply:
This hack was already in place and thus orthogonal to the purpose of this PR. Please let's move the discussion to its own issue  / PR.

Reply:
This is new code. Fix *this* pr please. The fix is trivial. I m sorry I did not suggest this before.